Detectors/Calibration/workflow/src/MeanVertexCalibratorSpec.cxx

```diff
@@ -0,0 +1,131 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @file   MeanVertexCalibratorSpec.cxx
+
+#include "Framework/ControlService.h"
+#include "Framework/ConfigParamRegistry.h"
+#include "Framework/Logger.h"
+#include "DetectorsCalibrationWorkflow/MeanVertexCalibratorSpec.h"
+#include "DetectorsCalibration/Utils.h"
+#include "CCDB/CcdbApi.h"
+#include "CCDB/CcdbObjectInfo.h"
+
+using namespace o2::framework;
+
+namespace o2
+{
+namespace calibration
+{
+void MeanVertexCalibDevice::init(InitContext& ic) {
+  
+    int minEnt = ic.options().get<int>("min-entries");
+    int nbX = ic.options().get<int>("nbinsX");
+    float rangeX = ic.options().get<float>("rangeX");
+    int nbY = ic.options().get<int>("nbinsY");
+    float rangeY = ic.options().get<float>("rangeY");
+    int nbZ = ic.options().get<int>("nbinsZ");
+    float rangeZ = ic.options().get<float>("rangeZ");
+    int nSlots4SMA = ic.options().get<float>("nSlots4SMA");
+    bool useFit = ic.options().get<bool>("useFit");
+    int slotL = ic.options().get<int>("tf-per-slot");
+    int delay = ic.options().get<int>("max-delay");
```

Comment:
I think this setting better to pass via ConfigurableObject, which will be eventually interfaced to ccdb.

Reply:
Ok, I will take a look on how to use it.

Reply:
Ok, I gave it a try, if you can check, it is this commit: 5361947b6e4bcfc633631de79a4ea05be43e1811